### PR TITLE
Update migration-controller SCC on OCP3.10/earlier (1.4.0)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1921559

CP for 4.5, 4.6, 4.7

### CODE REVIEW

File: modules/migration-upgrading-migration-tool-3.adoc [hunk 1]
```diff
@@ -70,13 +70,20 @@ $ oc -o yaml -n openshift-migration get deployment/migration-operator | grep ima
...
 
-. Create the `MigrationController` object:
+. Create the `migration-controller` object:
 +
 [source,terminal]
 ----
 $ oc create -f controller-3.yml
 ----
 
+. If your {product-title} version is 3.10 or earlier, set the security context constraint of the `migration-controller` to `anyuid` to enable direct image migration and direct volume migration:
```
Comment (shawn-hurley) [inline modules/migration-upgrading-migration-tool-3.adoc:80]: I think we should add that the migration-controller needs to be installed and that you are adding the security context constraint to the migration-controller service account?

File: modules/migration-upgrading-migration-tool-3.adoc [hunk 2]
```diff
@@ -70,13 +70,20 @@ $ oc -o yaml -n openshift-migration get deployment/migration-operator | grep ima
...
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user anyuid -Z migration-controller -n openshift-migration
```
Comment (shawn-hurley) [inline modules/migration-upgrading-migration-tool-3.adoc:84]: sub -Z with -z

**Review by shawn-hurley — COMMENTED** [review]
Wondering if `migration-controller` might be confusing so we should add that it is the service account?

**Comment by apinnick** [comment]
@shawn-hurley 
@XinRedhat said that the bug will probably be updated in the code of the 1.4.0 release, so this PR will be closed.

File: modules/migration-upgrading-migration-tool-3.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-upgrading-migration-tool-3.adoc:80]: The controller is created in the previous step. I can try to make the SA aspect clearer.

**Review by shawn-hurley — APPROVED** [review]

**Review by sergiordlr — APPROVED** [review]
Looks good to me.

Thank you very much!

**Review by jeana-redhat — APPROVED** [review]
LGTM

**Pull request merged**